app/scripts/reducers/granule-csv.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import {
+  GRANULE_CSV,
+  GRANULE_CSV_INFLIGHT,
+  GRANULE_CSV_ERROR
+} from '../actions/types';
+
+export const initialState = {
+  data: {},
```

Comment:
When data is received, it's just a string of text, correct? If so, I think it makes sense to default to `null`. Then you should also not need the `Object.entries(fileData).length === 0` above

Reply:
fixed